docs/general/web3-and-polkadot.md

```diff
@@ -0,0 +1,196 @@
+[Polkadot Vault](./polkadot-vault.md)). For **custodial** accounts, you trust third parties to
+manage your keys and give you access whenever needed.
+
+To sum up, data ownership comes from the fact that any message you sign with your private key comes
+from your digital identity, and the signature proof can be cryptographically verified. Unless
+someone else stole your identity, you and only you signed that message and own or are responsible
+for the information in it. Transferring an [NFT](./learn-nft-index) between two accounts is
+essentially a transfer of ownership.
+
+## Trustless Environment
+
+Cryptography also brings the possibility of building a trustless environment where we do not have to
+trust third parties, or have any relationship between the sender and receiver of a message. We do
+not need to trust centralized entities since we can verify who wrote the message and who owns what
+just by using cryptography. Trust is embedded in the code. Well-audited and reviewed code will
+ultimately provide a solid, trustless environment.
+
+## Data Immutability
+
+But what if the data we own can be easily modified or tampered with after they have been signed and
+stored?
+
+Here is where **blockchain** plays an important role. Blockchains are essentially databases where
+data are stored within blocks concatenated using hash functions, where, for example the hash of
+block `N + 1` contains data of that block together with the hash of the previous block `N`. This
+creates the situation where if you modify the content of block `N` you will change the hash of block
+`N + 1`, `N + 2`, etc. You will need to modify quite a lot of data, and in proof-of-stake
+blockchains like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} such attack is
+financially expensive, and attempting to do it will get you
+[slashed](../learn/learn-staking-advanced.md#slashing).
+
+So, with blockchain as a means of storing data permanently without any option to modify them, we can
+ensure that what we sign with our digital identity will not be modified.
+
+## Data Availability
+
+But what if our data are stored in a blockchain, but that blockchain is run on a centralized server
+or by different computers belonging to the same operator?
+
+That server or those computers can be easily shut down, the blockchain can be stopped from running
+and its data wiped out. This can be achieved from the inside by the malicious network participants
+or from the outside by regulatory rules and other forces. So, we would own our data, which would be
+immutable because stored on a blockchain, but that blockchain would be easily stopped. There would
+be little sense in owning something that can easily cease to exist in the future.
+
+Data availability is dependent on how resilient the blockchain is. Resiliency is achieved through
+decentralization, economic incentives, on-chain governance, and on-chain treasury funds to ensure
```

Comment:
I think on-chain governance might be enough to mention, as it implies that a treasury is needed in every context where a blockchain/decentralization is existant.

```suggestion
decentralization, economic incentives, and on-chain governance to ensure
```